examples/.pages

```diff
@@ -3,6 +3,7 @@ nav:
     - Docs folder: examples/ok-mkdocs-docs/docs
     - Ignore a folder: examples/ok-mkdocs-docs-ignore
     - Extra extensions: examples/ok-mkdocs-docs-extensions
+    - Custom extraction: examples/ok-mkdocs-custom-extract
```

Comment:
It looks like you're trying to maintain alphabetical order in the examples.

I was trying to keep a little bit of logical order in how examples were presented.  This probably belongs somewhere after line 11

Reply:
Moved as requested